python/agent.py

```diff
@@ -24,6 +25,14 @@ def __init__(self):
         self.ui_socket = None
         self.initialized = False
         self.modules = []
+        self.family_router = FamilyRouter()
+
+    def register_module(self, module):
+        self.modules.append(module)
+        self.family_router.register(module.FAMILY, module)
```

Comment:
I think we've made a past mistake here, and we are routing messages by just the family name, and not version or base DID. We should probably use module.BASE.

Reply:
That's a good point. I think I'll have to tweak the family router a little to support that.